### ISSUE TITLE
could you give more information about this tool?

### ISSUE THREAD

**Issue by cqzhao** [dogiv/MelWatch#1]
How could it be run?
What would it like when it runs, would it give any real-time plot?

**Comment by dogiv** [comment]
Yes, real-time plotting is the main purpose. You first create a config file (like the example TH_view.txt) that lists all the variables you want to plot. It also lists the filename of the plot file created by MELCOR (this has file extension .ptf and is named according to the MELCOR input file). 

You run MELCOR first so that it creates the plot file, then you run MelWatch from the command line, with the filename of the config file as a command line argument. e.g.:

python MelWatch.py example_config_file.txt

You will need to install python 3 and the imported packages (numpy, matplotlib, and tkinter I think). I wrote this a few years ago so probably I used python 3.5, but if I have time I'll test it with current versions of all those packages.
